### PR TITLE
updating base image for skipper Dockerfile

### PR DESCRIPTION
**Pull request by yevgeny-shnaidman**

### CODE REVIEW

File: Dockerfile.node-feature-discovery-build [hunk 1]
```diff
@@ -1,10 +1,10 @@
-FROM registry.ci.openshift.org/ocp/builder:rhel-8-golang-1.20-openshift-4.14
+FROM golang:1.20-alpine3.18
 
 ENV GO111MODULE=on
 ENV GOFLAGS=""
 
-RUN yum install -y which
-RUN yum install -y podman docker
+RUN ["apk", "add", "bash", "make", "docker", "curl", "shadow", "git"]
```
Comment (ybettan) [inline Dockerfile.node-feature-discovery-build:6]: Same questions as https://github.com/openshift/cluster-nfd-operator/pull/328#discussion_r1308466242

Comment (yevgeny-shnaidman) [reply Dockerfile.node-feature-discovery-build:6]: answered in [PR 328](https://github.com/openshift/cluster-nfd-operator/pull/328)

**Pull request merged**